examples/customEvent/CustomEventExample.js

```diff
@@ -0,0 +1,20 @@
+const { CustomEventConnector, Reshuffle } = require('../..')
+
+const app = new Reshuffle()
+const connector = new CustomEventConnector(app)
+
+connector.on({ channel: 'One', payLoad: { name: 'Jack', age: 25 } }, (event) => {
```

Comment:
channel seems like a new concept here, why not use something like `event` or 'eventTriggered'

Reply:
I think channel is to allow event grouping (like topic/queue name)